chapters/ch01.md

```diff
@@ -1,63 +1,92 @@
 # Why Machine Learning Strategy
-->
+-> 
 
+Tại sao cần chiến lược Machine Learning 
 
 Machine learning is the foundation of countless important applications, including web search, email anti-spam, speech recognition, product recommendations, and more. I assume that you or your team is working on a machine learning application, and that you want to make rapid progress. This book will help you do so.
 ->
 
+Machine Learning là nền tảng cho hàng loạt ứng dụng quan trọng như tìm kiếm trang web, lọc thư điện tử spam, nhận dạng giọng nói, gợi ý sản phẩm, và nhiều ứng dụng khác nữa. Nếu bạn cùng các thành viên trong nhóm đang làm một đề tài ứng dụng machine learning và rất muốn tiến triển nhanh chóng, thì quyển sách này là dành cho bạn.
+
+
 
 **Example: Building a cat picture startup**
 ->
 
+ Ví dụ: Khởi nghiệp ảnh mèo. 
```

Comment:
1. Nên dịch sát hơn là 'Xây dựng một công ty khởi nghiệp (hoặc giữ startup) về ảnh mèo'
2. Giữ format in đậm nhé cậu.
```
**Ví dụ: ...**
```